vendor/errors/v2.yml

```diff
@@ -839,9 +839,9 @@
   message: "Service broker returned dashboard client configuration without a dashboard URL"
 
 290001:
-  name: BuildpackNameTaken
-  http_code: 400
-  message: "The buildpack name is already in use: %s"
+  name: BuildpackNameStackTaken
+  http_code: 409
```

Comment:
Why did the http_code change?

Reply:
That seemed to us to be a better plan since the meaning changed. We are happy for you to leave it the same

Reply:
How did the meaning change?

Reply:
400 seemed wrong, it's meant for when the request could not be understood by the server.
409 seemed more appropriate, which indicates that there is a conflict with the resource.

Will changing this cause problems? If so we are happy to revert.

Reply:
I agree that 409 is more appropriate. Concern is that we break clients that rely on this behavior. @zrob thoughts?

Reply:
i'm not quite clear on what's happening from this snippet, but roughly, we should not change names/codes of existing errors.   if a different error is appropriate we should add a new entry and use that.  as far as 409 -- we have moved towards a convention of using 422 for this type of error and relying more on the text to describe the case than the code.